sormas-backend/src/test/java/de/symeda/sormas/backend/TestDatabaseCleaner.java

```diff
@@ -0,0 +1,36 @@
+package de.symeda.sormas.backend;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.persistence.Persistence;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.hilling.junit.cdi.jee.jpa.DatabaseCleaner;
+
+/**
+ * Cleanup database.
+ *
+ * <p>
+ * Brute force implementation.
+ * </p>
+ */
+public class TestDatabaseCleaner implements DatabaseCleaner {
+
+	protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+	public void run(Connection connection) throws SQLException {
+
+		connection.prepareStatement("DROP ALL OBJECTS").execute();
+		connection.commit();
+
+		Map<String, String> properties = new HashMap<>();
+		properties.put("hibernate.hbm2ddl.auto", "create");
```

Comment:
Use constants for better documentation (I used the `jakarta.persistence.schema-generation.database.action` definition here to be closer to JPA standard):

```
import org.hibernate.cfg.AvailableSettings;
import org.hibernate.tool.schema.Action;

		properties.put(AvailableSettings.HBM2DDL_DATABASE_ACTION, Action.CREATE.getExternalJpaName());
```

Reply:
Done. Except getExternalJpaName is not available on our current hibernate version.

Reply:
Okay, #11032 brings this version update: `org.hibernate:hibernate-core ............. 5.6.5.Final -> 5.6.14.Final`